### PR TITLE
Added multi tenant object databases

### PR DESCRIPTION
**Pull request by WilcoLouwerse**

### CODE REVIEW

File: api/src/Entity/Database.php [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+    /**
+     * @var Datetime The moment this resource was last Modified
+     *
+     * @Groups({"read"})
+     *
+     * @Gedmo\Timestampable(on="update")
+     *
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $dateModified;
```
Comment (rjzondervan) [inline api/src/Entity/Database.php]: Please make this DateTimeInterface-instances with type hint, see for example the Contract entity

File: api/src/Entity/Database.php [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+     *
+     * @ORM\Id
+     *
+     * @ORM\Column(type="uuid", unique=true)
+     *
+     * @ORM\GeneratedValue(strategy="CUSTOM")
+     *
+     * @ORM\CustomIdGenerator(class="Ramsey\Uuid\Doctrine\UuidGenerator")
+     */
+    private $id;
```
Comment (rjzondervan) [inline api/src/Entity/Database.php]: UuidInterface please :), see the File entity

**Review by rjzondervan — CHANGES_REQUESTED** [review]
Looks generally alright, but please add type hints _everywhere_

**Review by bbrands02 — DISMISSED** [review]

File: api/src/Entity/Database.php (hunk 1 above)
Comment (WilcoLouwerse) [reply api/src/Entity/Database.php]: will do

File: api/src/Entity/Database.php (hunk 2 above)
Comment (WilcoLouwerse) [reply api/src/Entity/Database.php]: will do

File: api/src/Entity/Database.php [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+    public function __construct()
+    {
+        $this->organizations = new ArrayCollection();
+    }
+
+    public function __toString(): string
+    {
+        return $this->getName();
+    }
```
Comment (rjzondervan) [inline api/src/Entity/Database.php]: You're missing a docblock here, and shouldn't this be _after_ the constructor

**Review by rjzondervan — CHANGES_REQUESTED** [review]

**Review by rjzondervan — APPROVED** [review]

**Pull request merged**